projectile.el

```diff
@@ -3427,7 +3427,10 @@ regular expression."
            (tags-exclude (projectile-tags-exclude-patterns))
            (default-directory project-root)
            (tags-file (expand-file-name projectile-tags-file-name))
-           (command (format projectile-tags-command tags-file tags-exclude default-directory))
+           (command (format projectile-tags-command
+                            tags-file
+                            tags-exclude
+                            (directory-file-name default-directory)))
```

Comment:
You should add some comment why this is needed here, otherwise someone might optimize it away. :-)